calcom/cal.com
# Add password on confirmation when changing email

## What does this PR do?



This PR prompts the user to confirm their password before changing their email.

Fixes # (issue)



https://www.loom.com/share/314277d527d0453a894f1d41741ce248

**Environment**: Staging(main branch) / Production

## Type of change





## How should this be tested?



- Go to the profile settings
- Try changing your email
	- In the prompt enter  the incorrect password
	- Now enter the correct password
- You should now be able to change your email

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
